wiki/People/Project_Loved_Team/en.md

```diff
@@ -0,0 +1,47 @@
+# Project Loved Team
+
+The **Project Loved Team** are members of the osu! community who manage [Project Loved](/wiki/Project_Loved), the voting platform that promotes beatmaps to the [Loved](/wiki/Beatmaps#loved) category.
+
+## Roles and Responsibilities
+
+Most of the Project Loved Team consists of **captains** for each of the [Game Modes](/wiki/Game_Modes). Captains are responsible for nominating beatmaps for the Loved category. They also write comments about the nominated maps to help readers get a feel for each map in case they have not seen or played them before.
```

Comment:
```suggestion
Project Loved Team mostly consists of **captains** for each [Game Mode](/wiki/Game_Modes). Captains are responsible for nominating beatmaps to the Loved category. They also write short descriptions about the nominated maps to help readers get a feel for each map in case they have not seen or played them before.
```

Reply:
can u add "The " at the front